### ISSUE TITLE
Unhandled exception when processing inline anotations

### ISSUE THREAD

**Issue by wallunit** [jsdoc3/jsdoc#972]
JsDoc 3.3.0-beta3 (as well as current master => 3c1f04396b990788abc38a51b0efdf16a95dddb9) crashes with the exception below when parsing the inline type annotations in following code:

```
var ns = {
  foo: function(/** Number */ a, /** Number */ b) {}
};
```

```
/usr/local/lib/node_modules/jsdoc/lib/jsdoc/src/visitor.js:101
            if ( !param || i === knownParams.indexOf(e.doclet.name) ) {
                                             ^
TypeError: Cannot call method 'indexOf' of undefined
    at null.<anonymous> (/usr/local/lib/node_modules/jsdoc/lib/jsdoc/src/visitor.js:101:46)
    at Visitor.visitNode (/usr/local/lib/node_modules/jsdoc/lib/jsdoc/src/visitor.js:411:24)
    at Visitor.visit (/usr/local/lib/node_modules/jsdoc/lib/jsdoc/src/visitor.js:314:27)
    at Walker.recurse (/usr/local/lib/node_modules/jsdoc/lib/jsdoc/src/walker.js:546:44)
    at Parser._walkAst (/usr/local/lib/node_modules/jsdoc/lib/jsdoc/src/parser.js:245:18)
    at Parser._parseSourceCode (/usr/local/lib/node_modules/jsdoc/lib/jsdoc/src/parser.js:236:18)
    at Parser.parse (/usr/local/lib/node_modules/jsdoc/lib/jsdoc/src/parser.js:157:18)
    at Object.module.exports.cli.parseFiles (/usr/local/lib/node_modules/jsdoc/cli.js:359:42)
    at module.exports.cli.main (/usr/local/lib/node_modules/jsdoc/cli.js:229:14)
    at Object.module.exports.cli.runCommand (/usr/local/lib/node_modules/jsdoc/cli.js:183:5)
    at /usr/local/lib/node_modules/jsdoc/jsdoc.js:134:13
    at Object.<anonymous> (/usr/local/lib/node_modules/jsdoc/jsdoc.js:136:3)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

**Comment by hegemonic** [comment]
Interestingly, we only crash if there are at least two params, both of which have comments.

I'll investigate.

**Comment by hegemonic** [comment]
Fixed on master and releases/3.3.
